src/main/webapp/app/exercises/modeling/assess/modeling-assessment-editor/modeling-assessment-editor.component.ts

```diff
@@ -193,7 +194,6 @@ export class ModelingAssessmentEditorComponent implements OnInit {
             this.alertService.info('modelingAssessmentEditor.messages.lock');
         }
         this.checkPermissions();
-        this.validateFeedback();
```

Comment:
Why don't we need this validation anymore (or was it not needed in the first place)?

Reply:
![image](https://user-images.githubusercontent.com/67344158/152235693-594b612a-2eba-4433-a31b-1d772e16bab0.png)
![image](https://user-images.githubusercontent.com/67344158/152235757-21bdcbb9-2713-4d4c-aa23-56edd1cd4392.png)

I believe it was not intended in the first place. The issue originally stated to solve the problem by implementing in accordance to the text exercises, where the override assessment button is already disabled. I checked that on page load even though the validation is not called, variables that are changed by the function are correct anyways. Also the correction round tag call is made before and after the validation so I believe we do not have any functionality loss here.